roadmap/implementers-guide/src/node/validity/approvals.md

```diff
@@ -0,0 +1,49 @@
+# Approval subsystem
```

Comment:
Can you add this file to the `SUMMARY.md` which constructs the book? Is this the approvals subsystem or the approval subsystem? The file name should reflect that.

Reply:
db2a8eec

I'll place them under validity for now, but I've no idea where these go really, nor even if disputes and approvals go into the same crate, although I suspect no..